package.json

```diff
@@ -4,6 +4,7 @@
   "description": "This is a bs port of the ocaml-containers project",
   "main": "index.js",
   "scripts": {
+    "start": "bsb -w",
```

Comment:
`start` is an actual `npm` command while watch is not, you have to do `npm run watch`, so I prefer `start`